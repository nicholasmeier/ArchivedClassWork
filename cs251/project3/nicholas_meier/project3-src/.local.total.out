CS251: . nmeier Sat Feb 10 16:09:50 EST 2018  Total:  6 of 10
CS251: . nmeier Mon Feb 12 21:54:05 EST 2018  Total:  0 of 0
CS251: . nmeier Mon Feb 12 21:54:10 EST 2018  Total:  6 of 10
CS251: . nmeier Tue Feb 13 15:32:59 EST 2018  Total:  0 of 0
CS251: . nmeier Tue Feb 13 15:33:09 EST 2018  Total:  0 of 0
CS251: . nmeier Tue Feb 13 15:33:17 EST 2018  Total:  6 of 10
CS251: . nmeier Tue Feb 13 16:13:03 EST 2018  Total:  6 of 10
CS251: . nmeier Tue Feb 13 16:13:38 EST 2018  Total:  6 of 10
CS251: . nmeier Tue Feb 13 16:29:23 EST 2018  Total:  6 of 10
CS251: . nmeier Wed Feb 21 23:07:53 EST 2018  Total:  0 of 10
