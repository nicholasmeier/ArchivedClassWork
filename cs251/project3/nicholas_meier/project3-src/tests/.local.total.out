CS251: . elkindi Sat Feb 3 14:34:17 EST 2018  Total:  140 of 140
CS251: . elkindi Sat Feb 3 14:37:44 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:37:50 EST 2018  Total:  8 of 8
CS251: . elkindi Sat Feb 3 14:39:37 EST 2018  Total:  8 of 8
CS251: . elkindi Sat Feb 3 14:41:21 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:42:17 EST 2018  Total:  80 of 80
CS251: . elkindi Sat Feb 3 14:42:25 EST 2018  Total:  100 of 100
CS251: . elkindi Sat Feb 3 14:58:29 EST 2018  Total:  0 of 88
CS251: . elkindi Sat Feb 3 14:59:29 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 14:59:34 EST 2018  Total:  80 of 88
CS251: . elkindi Sat Feb 3 14:59:50 EST 2018  Total:  80 of 88
CS251: . elkindi Sat Feb 3 15:00:57 EST 2018  Total:  88 of 88
CS251: . elkindi Sat Feb 3 15:02:53 EST 2018  Total:  112 of 112
CS251: . elkindi Sat Feb 3 15:05:14 EST 2018  Total:  80 of 80
CS251: . elkindi Sat Feb 3 15:05:33 EST 2018  Total:  100 of 100
CS251: . elkindi Sat Feb 3 15:12:14 EST 2018  Total:  2 of 10
CS251: . elkindi Sat Feb 3 15:15:46 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:16:11 EST 2018  Total:  10 of 10
CS251: . elkindi Sat Feb 3 15:17:42 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:18:05 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:20:07 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:25:12 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:31:16 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:32:21 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:33:13 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:33:59 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:43:43 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:43:48 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:45:15 EST 2018  Total:  0 of 80
CS251: . elkindi Sat Feb 3 15:48:47 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:48:49 EST 2018  Total:  0 of 10
CS251: . elkindi Sat Feb 3 15:48:52 EST 2018  Total:  0 of 80
CS251: . nmeier Sat Feb 10 15:48:40 EST 2018  Total:  0 of 0
CS251: . nmeier Tue Feb 13 16:29:42 EST 2018  Total:  1 of 10
CS251: . nmeier Tue Feb 13 16:45:00 EST 2018  Total:  1 of 10
CS251: . nmeier Tue Feb 13 16:45:12 EST 2018  Total:  1 of 10
CS251: . nmeier Tue Feb 13 16:45:30 EST 2018  Total:  0 of 0
CS251: . nmeier Tue Feb 13 16:47:04 EST 2018  Total:  1 of 10
CS251: . nmeier Tue Feb 13 16:48:20 EST 2018  Total:  1 of 10
CS251: . nmeier Tue Feb 13 16:52:13 EST 2018  Total:  1 of 10
CS251: . nmeier Tue Feb 13 16:53:00 EST 2018  Total:  1 of 100
CS251: . nmeier Tue Feb 13 16:53:09 EST 2018  Total:  1 of 10
CS251: . nmeier Wed Feb 14 15:49:05 EST 2018  Total:  1 of 10
CS251: . nmeier Wed Feb 14 15:49:59 EST 2018  Total:  1 of 10
CS251: . nmeier Wed Feb 14 15:52:01 EST 2018  Total:  4 of 10
CS251: . nmeier Wed Feb 14 16:20:21 EST 2018  Total:  4 of 10
CS251: . nmeier Fri Feb 16 14:21:29 EST 2018  Total:  3 of 10
CS251: . nmeier Fri Feb 16 14:23:31 EST 2018  Total:  3 of 10
CS251: . nmeier Fri Feb 16 14:33:56 EST 2018  Total:  3 of 10
CS251: . nmeier Fri Feb 16 14:40:18 EST 2018  Total:  6 of 10
CS251: . nmeier Fri Feb 16 14:44:20 EST 2018  Total:  6 of 10
CS251: . nmeier Fri Feb 16 14:46:17 EST 2018  Total:  6 of 10
CS251: . nmeier Fri Feb 16 14:50:09 EST 2018  Total:  6 of 10
CS251: . nmeier Fri Feb 16 14:54:21 EST 2018  Total:  6 of 10
CS251: . nmeier Fri Feb 16 14:57:11 EST 2018  Total:  10 of 10
CS251: . nmeier Fri Feb 16 15:18:19 EST 2018  Total:  0 of 10
CS251: . nmeier Fri Feb 16 15:19:03 EST 2018  Total:  0 of 10
CS251: . nmeier Fri Feb 16 19:33:34 EST 2018  Total:  0 of 10
CS251: . nmeier Mon Feb 19 08:47:15 EST 2018  Total:  0 of 10
CS251: . nmeier Mon Feb 19 08:47:33 EST 2018  Total:  0 of 10
CS251: . nmeier Mon Feb 19 08:47:52 EST 2018  Total:  10 of 10
CS251: . nmeier Mon Feb 19 08:50:17 EST 2018  Total:  0 of 10
CS251: . nmeier Wed Feb 21 23:05:27 EST 2018  Total:  0 of 0
CS251: . nmeier Wed Feb 21 23:08:12 EST 2018  Total:  0 of 10
CS251: . nmeier Sun Feb 25 21:53:52 EST 2018  Total:  0 of 10
CS251: . nmeier Mon Mar 26 23:23:12 EDT 2018  Total:  10 of 100
